core/server/data/timezones.json

```diff
@@ -2,328 +2,263 @@
 		},
 		{
 			"name": "America/Denver",
-			"label": "(GMT -7:00) Mountain Time (US & Canada)",
-			"offset": -420
+			"label": "(GMT -7:00) Mountain Time (US & Canada)"
 		},
 		{
 			"name": "America/Costa_Rica",
-			"label": "(GMT -6:00) Central America",
-			"offset": -360
+			"label": "(GMT -6:00) Central America"
 		},
 		{
 			"name": "America/Chicago",
-			"label": "(GMT -6:00) Central Time (US & Canada)",
-			"offset": -360
+			"label": "(GMT -6:00) Central Time (US & Canada)"
 		},
 		{
 			"name": "America/Mexico_City",
-			"label": "(GMT -6:00) Guadalajara, Mexico City, Monterrey",
-			"offset": -360
+			"label": "(GMT -6:00) Guadalajara, Mexico City, Monterrey"
 		},
 		{
 			"name": "America/Regina",
-			"label": "(GMT -6:00) Saskatchewan",
-			"offset": -360
+			"label": "(GMT -6:00) Saskatchewan"
 		},
 		{
 			"name": "America/Bogota",
-			"label": "(GMT -5:00) Bogota, Lima, Quito",
-			"offset": -300
+			"label": "(GMT -5:00) Bogota, Lima, Quito"
 		},
 		{
 			"name": "America/New_York",
-			"label": "(GMT -5:00) Eastern Time (US & Canada)",
-			"offset": -300
+			"label": "(GMT -5:00) Eastern Time (US & Canada)"
 		},
 		{
 			"name": "America/Fort_Wayne",
-			"label": "(GMT -5:00) Indiana (East)",
-			"offset": -300
+			"label": "(GMT -5:00) Indiana (East)"
 		},
 		{
 			"name": "America/Caracas",
-			"label": "(GMT -4:30) Caracas, La Paz",
-			"offset": -270
+			"label": "(GMT -4:00) Caracas, La Paz"
 		},
 		{
 			"name": "America/Halifax",
-			"label": "(GMT -4:00) Atlantic Time (Canada); Brasilia, Greenland",
-			"offset": -240
+			"label": "(GMT -4:00) Atlantic Time (Canada); Brasilia, Greenland"
 		},
+        {
+            "name": "America/Santiago",
+            "label": "(GMT -4:00) Santiago"
+        },
 		{
 			"name": "America/St_Johns",
-			"label": "(GMT -3:30) Newfoundland",
-			"offset": -210
+			"label": "(GMT -3:30) Newfoundland"
 		},
 		{
 			"name": "America/Argentina/Buenos_Aires",
-			"label": "(GMT -3:00) Buenos Aires, Georgetown",
-			"offset": -180
-		},
-		{
-			"name": "America/Santiago",
-			"label": "(GMT -3:00) Santiago",
-			"offset": -180
+			"label": "(GMT -3:00) Buenos Aires, Georgetown"
 		},
 		{
 			"name": "America/Noronha",
-			"label": "(GMT -2:00) Fernando de Noronha",
-			"offset": -120
+			"label": "(GMT -2:00) Fernando de Noronha"
 		},
 		{
 			"name": "Atlantic/Azores",
-			"label": "(GMT -1:00) Azores",
-			"offset": -60
+			"label": "(GMT -1:00) Azores"
 		},
 		{
 			"name": "Atlantic/Cape_Verde",
-			"label": "(GMT -1:00) Cape Verde Is.",
-			"offset": -60
+			"label": "(GMT -1:00) Cape Verde Is."
 		},
 		{
 			"name": "Africa/Casablanca",
-			"label": "(GMT) Casablanca, Monrovia",
-			"offset": 0
-		},
+			"label": "(GMT) Casablanca, Monrovia"
+        },
 		{
 			"name": "Europe/Dublin",
-			"label": "(GMT) Greenwich Mean Time : Dublin, Edinburgh, London",
-			"offset": 0
-		},
+			"label": "(GMT) Greenwich Mean Time : Dublin, Edinburgh, London"
```

Comment:
Agree with @kevinansfield because this is the only occurrence of a named timezone in the list.  Timezone names change in some areas `GMT+1` would correctly be BST (British Summer Time) and then it would also make sense to add CET vs. CEST ...